File: Python/program73.py
class Digit:
    _No = 0
    
    def __init__(self, iNo):
        self._No = iNo
           
    def Reverese(self):
        
        if(self._No < 0):
            self._No = -self._No
            
        iRev = 0
        while(self._No > 0):
            
            iDigit = self._No % 10
            iRev = iRev * 10 + iDigit
            self._No = self._No // 10 
            
        return iRev
    
def main(): 
    
    iValue = int (input("Enter number : "))
        
    sobj = Digit(iValue)
    iRet = sobj.Reverese()
    
    print("Reverse number is ",iRet)
    
if __name__ == "__main__":
    main()

File: Java/program73.java
import java.util.*;

class Digits
{
    public int Reverse(int iNo)
    {
        int iCnt = 0, iDigit = 0, iRev = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }
        while(iNo > 0) 
        {
            iDigit = iNo % 10;
            iRev = (iRev *10) + iDigit;
            iNo = iNo /10;
        }

        return iRev;

      
    }

}
class program73
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iRet = 0;

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Digits dobj =new Digits();

        iRet =dobj.Reverse(iValue);

         System.out.println("Reverse number is  : "+iRet);

       

    }
}